Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'http://rubygems.org'
 
-gem 'rake', '~> 10.4'
+gem 'rake'
```

Comment:
Rake 10 doesn't work well with Ruby 3.2. Loosening this version constraint gets the 3.2 and Ruby head steps working.